lib/index.coffee

```diff
@@ -53,16 +56,16 @@ module.exports = class ReserveResource
         cb_wf err, lock_status
     ], cb
 
-  wait_until_lock: (resource, cb) ->
-    # waits until a lock on the specific resource can be obtained
+  wait_until_lock: (system_id, cb) ->
```

Comment:
I think we should keep this as `resource` everywhere since this specific repo/library is intended to be general-purpose and is also open-source, and `system_id` only makes sense for our primary internal use case.